File: bubblesort.js
function bubbleSort(arr){
    let n = arr.length
    for (i=0;i<n;i++){
      for (j=0;j<(n-i-1);j++){
       if (arr[j] > arr [j+1]){
         var temp = arr[j]
         arr[j] = arr[j + 1]
         arr[j + 1] = temp
       }
      }
    }
 }

 let arr = [];
 for (let i = 0; i < 10; i++) { 
    arr.push(Math.floor(Math.random() * 100)); 
 }
 
 bubbleSort(arr)
 
 console.log(" Sorted array is: ")

 console.log(arr.join(' '));
 

File: bubbleSort.py
def bubbleSort(arr):
   n = len(arr)
   for i in range (n):
     for j in range (0, n-i-1):
      if arr[j] > arr [j+1] :
        arr[j], arr[j+1] = arr[j+1], arr[j]

arr = [ 23, 42, 435, 334, 23, 12, 246, 1, 34, 9, 98, 4]

bubbleSort(arr)

print(" Sorted array is: ")

for i in range(len(arr)):
   print(arr[i], end = ' ')